src/jvm.rs

```diff
@@ -105,12 +106,13 @@ pub trait IsVoid: Default {}
 impl IsVoid for () {}
 
 static GLOBAL_JVM: Lazy<JavaVM> = Lazy::new(|| {
-    let jvm_args = InitArgsBuilder::new()
+    let mut jvm_builder = InitArgsBuilder::new()
         .version(jni::JNIVersion::V8)
-        .option("-Xcheck:jni")
-        .option("-Djava.class.path=java")
-        .build()
-        .unwrap();
+        .option("-Xcheck:jni");
+    if let Ok(classpath) = env::var("CLASSPATH") {
+        jvm_builder = jvm_builder.option(format!("-Djava.class.path={classpath}"));
+    }
+    let jvm_args = jvm_builder.build().unwrap();
 
     JavaVM::new(jvm_args).unwrap()
 });
```

Comment:
I just noticed that this is not the nice `Jvm::builder().add_classpath(..).launch(|jvm| ...)` described in the documentation.

Reply:
I find the builder pattern to be a bit strange, because it's not possible to build more than one JVM per process. What about changing the type of `GLOBAL_JVM` to be `OnceLock<JavaVM>`, so that the following is possible?
```rust
Jvm::builder().add_classpath(..).launch()?;
Jvm::with(|jvm| ...)?;
Jvm::with(|jvm| ...)?;
Jvm::with(|jvm| ...)?;
```
Calling `Jvm::builder().launch()` on an already initialized JVM would return an error, as well as calling `Jvm::with(..)` on a non-initialized JVM.

Reply:
I don't know what a "once lock" is, but that is kind of what I had in mind -- except that I figured we would have a default so if you called `Jvm::with` without having invoked `launch`, you got a default configuration.

Reply:
The one constraint I want to maintain is that you have at most one `Jvm` handle available at a time -- i.e., nested `Jvm::with` should not work. I'm not sure how important this is, but it lets us do nested frames if we ever want that.

Reply:
let's follow up on https://github.com/duchess-rs/duchess/issues/6